SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -435,6 +435,11 @@ void USpatialReceiver::RemoveActor(Worker_EntityId EntityId)
 		return;
 	}
 
+	if (Actor->GetTearOff())
+	{
+		return;
```

Comment:
There's additional logic we need to do here.
1. Clean up the actor channel (this might happen automatically when the tear off is issued, but worth verifying)
2. Remove the entity id reference (CleanupDeletedEntity(...))

Reply:
Added cleanup stuff now 👍